Test/git-issues/git-issue-3410/git-issue-3410a.dfy

```diff
@@ -0,0 +1,10 @@
+// RUN: %baredafny run --no-verify --use-basename-for-filename "%s" > "%t"
```

Comment:
Rather than just `--use-basename-for-filename`, most tests that use `%baredafny` use `%args` instead.

Reply:
Ah. And the reason I forgot to use it is that %args does not play well with 'run' and other such verbs